awx/ui_next/src/screens/Inventory/InventoryHostAdd/InventoryHostAdd.jsx

```diff
@@ -1,8 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
 import { CardBody } from '@patternfly/react-core';
+import InventoryHostForm from '../shared/InventoryHostForm';
+import { InventoriesAPI } from '@api';
 
 function InventoryHostAdd() {
-  return <CardBody>Coming soon :)</CardBody>;
+  const [formError, setFormError] = useState(null);
+  const history = useHistory();
+  const { id } = useParams();
+
+  const handleSubmit = async values => {
+    try {
+      const { data: response } = await InventoriesAPI.createHost(id, values);
+      history.push(`/inventories/inventory/${id}/hosts/${response.id}/details`);
```

Comment:
@marshmalien Would it be better for this to take you back to the related _hosts_  tab instead of the details tab? I could see this being problematic for users who want to add several hosts in a row.

Reply:
Redirecting to the details view in the current UX flow, but I see what you mean. Do you think this is something we should only do for Inventory Hosts?

Reply:
https://github.com/ansible/awx/pull/5491#issuecomment-565594337
@trahman73 @marshmalien :+1: works for me